package.json

```
@@ -52,25 +52,24 @@
     "husky": "^4.3.8",
     "jsdoc": "^3.6.6",
     "lint-staged": "^8.0.0",
-    "node-sass-chokidar": "^1.5.0",
-    "npm-run-all": ">=4.1.2",
+    "node-sass": "^5.0.0",
     "prettier": "^2.2.1",
     "react-scripts": "^4.0.3",
     "stylelint": "^13.11.0",
     "stylelint-config-recommended-scss": ">=3.0.0",
     "stylelint-scss": "^3.19.0"
   },
   "scripts": {
-    "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch",
+    "build-css": "node-sass src/ -o src/",
+    "watch-css": "npm run build-css && node-sass src/ -o src/ --watch",
     "start-js": "react-scripts start",
-    "start": "npm-run-all --parallel watch-css start-js",
+    "start": "npm run watch-css & npm run start-js",
```

Typo here, causes `npm start` to freeze hang after running `watch-css`
```suggestion
    "start": "npm run watch-css && npm run start-js",
```